.rubocop.yml

```diff
@@ -271,6 +271,9 @@ Style/HashAsLastArrayItem:
 Style/HashLikeCase:
   Enabled: true
 
+Style/HashSyntax:
+  EnforcedShorthandSyntax: never
```

Comment:
I prefer setting this to `either`. We should allow folks to use which ever syntax seems appropriate for the task.